Optimizing the Security of your Skywallet.md

```diff
@@ -2,9 +2,9 @@
 
 Even though the Skywallet provides greater security when compared to a Desktop Wallet, it is essential to follow these security optimizations to get maximum security benefit from your Skywallet.
 
-Two things are necessary for a third party to access your funds in Skywallet, which is either the PIN code of the Skywallet or the Seed of the Skywallet.
+One of two things is necessary for a third party to access the Skywallet, which is either the PIN code of the Skywallet and the Seed of the Skywallet.
 
-You are recommended to create a secure PIN code, although, unless your Skywallet is lost or got stolen, no one can access it using the PIN code.
+It is recommended to create a secure PIN code, although, unless your Skywallet is lost or got stolen, no one can access it using the PIN code.
```

Comment:
Reading it again, it is best to just say to use a secure PIN. The second part of the sentence might disincentivize people to setup a secure PIN.